server/src/main/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceService.java

```diff
@@ -253,7 +262,7 @@ boolean updateDesiredBalanceAndReroute(DesiredBalanceInput desiredBalanceInput,
                     final var newNodes = desiredAssignment.getValue();
                     final var oldNodes = desiredBalance.desiredAssignments().get(shardId);
                     if (newNodes.equals(oldNodes)) {
-                        logger.trace("{} desired balance unchanged,   allocating to {}", shardId, newNodes);
+                        logger.trace("{} desired balance unchanged, allocating to {}", shardId, newNodes);
```

Comment:
The extra spaces were to keep it lined up with the `now allocating` message in the `else` branch in the logs. I'm ok to remove them, but they seemed helpful sometimes.